src/categorical_algebra/DataMigrations.jl

```diff
@@ -161,6 +162,27 @@ function migrate!(X::ACSet, Y::ACSet, FOb, FHom)
   migrate!(X, Y, M)
 end
 
+
+# On morphisms
+
+migrate(t::Type{T}, f::ACSetTransformation, M::DeltaMigration) where T <: ACSet = 
+  migrate(t, f, M.functor)
```

Comment:
This will probably be able to be just any kind of migration, but maybe let's not worry about that until datamigrations.jl comes out.